### DISCUSSION TITLE
"The current file is restricted and won't be used as context" - as  `Copilot Individual` user - Why?

### DISCUSSION THREAD

**Discussion by mrclrchtr** [community/community#128991] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As in the description: I'm an individual user and getting the following message:

![image](https://github.com/community/community/assets/16151098/3bfafccb-b04a-43d4-8902-7a10c408e150)

In the description here: https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/configuring-content-exclusions-for-github-copilot it is not a feature, that I can manage as individual, right?

So, whats up?

**Comment by wrq9** [comment]
same issue

**Comment by Gumbraise** [comment]
It's because of the last update of Copilot. Bugged version dw

**Comment by rothwerx** [comment]
This is happening to me, also as an individual user, and my repo is not hosted in Github at all.

**Comment by aszaw** [comment]
getting the same issue in android studio as an induvidual user, what ide are you using?

**Comment by andretcarpizo** [comment]
Same issue in JetBrains Rider

![image](https://github.com/community/community/assets/69943379/6545a54a-1b1f-417c-8452-6fbc3f76a0cb)

**Comment by martinwangen** [comment]
Same with me, worked before the update

**Comment by PatZab** [comment]
yapp same here

**Comment by lloydchandran** [comment]
same in goland

**Comment by GatoDev** [comment]
Another IntellJ Copilot error… I don't know who is in charge of this plugin but what a disaster. Not to mention how behind they are with visual studio. Start adding features at once and not bugs...

**Comment by Cassiecassie123** [comment]
same

**Comment by DaoyuanLi2816** [comment]
yeah same

**Comment by JinxuanChen03** [comment]
same after recent update

**Comment by hstaudacher** [comment]
We already fixed the issue in the nightly version and submitted 1.5.10.5840 to the JetBrains marketplace. It will be available shortly. Sorry about the inconvenience.

**Reply by GatoDev** [reply]
One bug fix, one added.

![image](https://github.com/community/community/assets/36801628/69742f2e-16ea-4e57-b932-08d48fed04a4)

**Comment by trajan0x** [comment]
+!, also seeing this

**Comment by lucychen0103** [comment]
same issue

**Comment by krisraven** [comment]
Getting this issue in Webstorm IDE.

I can ask Copilot something about the code, but then any follow up question seems to be blocked. So I need to start a new chat to ask the follow up question.

**Reply by mrclrchtr** [reply]
I have the feeling that follow-up questions are currently completely broken...
